tests/python_functional/src/driver_io/file/file.py

```diff
@@ -52,3 +58,7 @@ def wait_for_creation(self):
     def open_file(self, mode):
         self.__opened_file = open_file(self.__file_path, mode)
         return self.__opened_file
+
+    def open_named_pipe(self):
```

Comment:
As this method is analog with `open_file()` and the parent of `PipeIO` is the `File` as also in `FileIO` case, I think better place is in the `File` class.
In this case I prefer the consistency with `FileIO` class

Reply:
`open_file`was put into `File` because at that time of writing, `File` was considered a final class. Now we have different file objects: `raw file` and `pipe`. This is why `File` is made into and interface in this PR.
The clean followup would be to create a `RawFile` class, and put `open_file` there.

That maybe result in a nearly empty class, so I do not want to push that. My suggestion is: there should only be an `open` method, (the default implementation can be the same as open_file), which is overridden by the implementation of `open_named_pipe`. And neither `open_file`, nor `open_named_pipe` should be seen in the `File` interface.